docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -378,8 +378,8 @@ POST _ml/data_frame/_evaluate
 
 <1> The destination index which is the output of the {dfanalytics-job}.
 <2> Calculate the training error by evaluating only the training data.
-<3> The field that contains the actual (ground truth) value.
-<4> The field that contains the predicted value.
+<3> Field containing the actual (ground truth) value.
```

Comment:
> Field containing

I'm not sure I understand how this change is an improvement.

Reply:
@lcawl I tried to achieve shorter sentences with less complicated structures. I thought that this way the sentence is less conversational. Do you prefer to change it back?